memebot/memebot.py

```diff
@@ -63,6 +63,9 @@ def __init__(self, *, known_memes: Iterable[Meme], guild_config: str):
         self.commands['!skip'] = create_skip_executor(self.guild_player_config.get)
         self.commands['!queue'] = create_show_queue_executor(self.guild_player_config.get)
         self.commands['!remove'] = create_queue_remove_executor(self.guild_player_config.get)
+        now_playing_executor = create_now_playing_executor(self.guild_player_config.get)
+        self.commands['!now playing'] = now_playing_executor
+        self.commands['!np'] = now_playing_executor
```

Comment:
Do you think we should find a way to refactor this so that maybe there's a setup function for aliases for each command? I've thought about wanting `!q` for the queue.

Reply:
I agree but haven't really come up with a good way to do it. Open to suggestions.